packages/shared/src/features/onboarding/steps/FunnelRegistration.tsx

```diff
@@ -27,21 +27,16 @@ import { useLogContext } from '../../../contexts/LogContext';
 import { broadcastChannel } from '../../../lib/constants';
 import Logo, { LogoPosition } from '../../../components/Logo';
 import type { LoggedUser } from '../../../lib/user';
-import type { FunnelStepTransitionCallback } from '../types/funnel';
 import { FunnelStepTransitionType } from '../types/funnel';
 import { sanitizeMessage } from '../shared';
-
-interface FunnelRegistrationProps {
-  onTransition?: FunnelStepTransitionCallback<void>;
-  heading: string;
-  imageMobile: string;
-  image: string;
-}
+import type { FunnelStepSignup } from '../types/funnel';
```

Comment:
I feel like we need some comments here for our future selfs @sshanzel

Reply:
AuthOption in the making 😅

Reply:
Did a bit of refactoring here 🙏 

https://github.com/dailydotdev/apps/pull/4396